WSM-EP-DP/events/WestmereEP-DP_core.json

```diff
@@ -878,8 +878,8 @@
       "EventCode": "0x49",
       "UMask": "0x20",
       "EventName": "DTLB_MISSES.PDE_MISS",
-      "BriefDescription": "DTLB misses casued by low part of address",
-      "PublicDescription": "DTLB misses casued by low part of address",
+      "BriefDescription": "DTLB misses caused by low part of address",
```

Comment:
Looks good to me. Please increase the version in the header at the top of this file.